File: app/src/main/java/com/example/stiffback/remoteDataSource/ElevationValue.java
package com.example.stiffback.remoteDataSource;

import com.google.gson.annotations.SerializedName;

public class ElevationValue {
    @SerializedName("USGS_Elevation_Point_Query_Service")
    public PointQueryService mPointQueryService;

    public class PointQueryService{
        @SerializedName("Elevation_Query")
        public ElevationQuery mElevationQuery;

        public class ElevationQuery{
            @SerializedName("x")
            public Double mLng;
            @SerializedName("y")
            public Double mLat;
            @SerializedName("Data_Source")
            public String mDataSource;
            @SerializedName("Elevation")
            public Double mElevation;
            @SerializedName("Units")
            public String mUnits;
        }
    }
}

File: app/src/main/java/com/example/stiffback/remoteDataSource/ElevationValue.kt
package com.example.stiffback.remoteDataSource

import com.google.gson.annotations.SerializedName

class ElevationValue {
    @SerializedName("USGS_Elevation_Point_Query_Service")
    lateinit var mPointQueryService: PointQueryService

    inner class PointQueryService {
        @SerializedName("Elevation_Query")
        lateinit var mElevationQuery: ElevationQuery

        inner class ElevationQuery {
            @SerializedName("x")
            var mLng: Double? = null
            @SerializedName("y")
            var mLat: Double? = null
            @SerializedName("Data_Source")
            var mDataSource: String? = null
            @SerializedName("Elevation")
            var mElevation: Double? = null
            @SerializedName("Units")
            var mUnits: String? = null
        }
    }
}